README.md

```diff
@@ -63,6 +63,14 @@ Download the latest A-4E-C installation file from the [releases page](https://gi
 
 Unzip the contents of the zip file into the following folder: `C:\Users\username\Saved Games\DCS`
 
+**NOTE: The mod MUST be installed into this folder, EVEN if you have installed DCS into a different folder than default.**
+
+**FAILURE TO DO SO WILL CAUSE THE MOD TO NOT WORK AND WILL CAUSE YOUR CLIENT TO FAIL THE IC CHECK**
+
+If you are installing onto an openbeta client the path changes to: `C:\Users\username\Saved Games\DCS.openbeta`
+
+Note: The above does not apply to steam clients, they are still installed into: `C:\Users\username\Saved Games\DCS`
+
 _To assist JSGME users, we’ve included the “mods/aircraft/” folders in the zip. You can safely merge these folders with your DCS World user folder._
```

Comment:
I think OvGME would be a better here. I feel like it's prefered over JSGME and is more up to date and manages backups better.

Reply:
I don't use mod managers so had no experience and didn't know the preferred method. I'll change it from "JSGME users" to "OvGME and JSGME users"

Reply:
Yes you are right since it's installed to Saved Games, there is really no need for mod manager. But old habits die hard on some people.

Reply:
hola sabes q cuando abro el juego dcs me aparece un cartel que dice que nbo esta abilitado y se desactivara, q ouede ser?